gdk4/src/auto/display.rs

```diff
@@ -175,22 +175,8 @@ impl Display {
         }
     }
 
-    pub fn get_property_composited(&self) -> bool {
```

Comment:
And this one

Reply:
Same as... you got the idea :) https://github.com/gtk-rs/gtk4-rs/blob/8107dfccc9c8a3939f1eaf45b23911dfc05d986e/gdk4/src/auto/display.rs#L134